src/include/txn_inline.h

```diff
@@ -945,7 +945,7 @@ __wt_txn_visible(
 static inline WT_VISIBLE_TYPE
 __wt_txn_upd_visible_type(WT_SESSION_IMPL *session, WT_UPDATE *upd)
 {
-    uint8_t prepare_state;
+    uint8_t prepare_state, previous_state;
```

Comment:
Is this file supposed to be impacted?

Reply:
No, it's left over from something else. I need to remove this commit somehow.

Reply:
Ah I see, I spoiled my local `develop` branch... Fixed it by rebasing ontop of origin/develop.